docs/source/deployguide/ca-deploy-topology.md

```diff
@@ -0,0 +1,58 @@
+
+Another reason to include an intermediate CA would be when you have a very large organization with multiple departments and you don’t want a single CA to generate certificates for all of the departments. Intermediate CAs provide a mechanism for scoping the certificates that a CA manages to a smaller department or sub-group. Intermediate CAs are not required, but they mitigate risk and scope certificate management. This pattern incurs significant overheard if there will only be a small number of members in the organization. As an alternative to deploying multiple intermediate CAs, you can configure a CA with `affiliations` (similar to departments) instead. More on this later when we talk about `affiliations`.
+
+Also, in situations where it is acceptable for one TLS CA to be used to issue certificates to secure communications for an entire organization, it would be reasonable for intermediate CAs to use the same TLS CA as the root CA rather than having their own dedicated TLS CA.
+
+### In what order should the CAs be deployed?
+
+If a dual-headed CA is not configured with an enrollment CA and TLS CA, then the TLS CA is deployed separately, and needs to be deployed _before_ the enrollment CA in order to generate the TLS certificate for the enrollment CA. After the TLS CA is deployed, then the enrollment CA can be deployed followed by any intermediate CAs if required.
+
+## A note about configuration
+
+There are three ways to configure settings on the Fabric CA server and client. The precedence order for overriding the default settings is:
+
+1. Use the [Fabric CA server CLI commands](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html).
+2. Use environment variables to override configuration file settings.
+3. Modify the configuration file.
+
+However, the recommendation is that before starting the server, you **make the modifications to the configuration file** to become familiar with the available settings and how they work.  Note that some configuration settings are stored in the CA database; therefore, after a CA is started, overriding the settings can no longer be performed by editing the configuration file or by setting environment variables. Affected parameters are noted throughout these instructions. In these cases, the modifications are required to be made by using the Fabric CA server CLI commands and have the added benefit of not requiring a server restart. 
```

Comment:
This gets a little tricky if deploying on Kubernetes instead of Docker.  We should think about documenting the environment variable equivalents to assist in k8s deployments as environment variables will most likely be used in the k8s deployment config versus 'fabric-ca-server init' and then modifying the config file.

Reply:
@denali49 
Could you open a jira issue and start dropping the specifics in there?

We can decide if we want to doc these kinds of considerations i
- inline here 
- or a separate considerations topic on this page 
- or a totally separate page once we see the volume of differences. 

What do you think?

Reply:
We will address this either via an additional topic or inline content once the details are well defined.